to_shmem/Cargo.toml

```diff
@@ -1,7 +1,9 @@
 [package]
-name = "to_shmem"
+name = "stylo_to_shmem"
```

Comment:
Why are you renaming `to_shmem` and `to_shmem_derive` here?

Reply:
This PR represents exactly what was published for the 0.1.0 release, for which I had to rename any crate that had already been published (as I didn't have publish access to publish a new version). It will need to look a little different for actually upstreaming. I just thought it would be good to have a reference of what was actually published (probably we should turn this into a branch in this repo).